build/webpack/webpack.extension.node.config.js

```diff
@@ -120,6 +120,14 @@ const config = {
                 }
             ]
         }),
+        new copyWebpackPlugin({
+            patterns: [
+                {
+                    from: './node_modules/jquery/dist/jquery.min.js',
+                    to: './node_modules/jquery/dist/jquery.min.js'
```

Comment:
source and destination are the same?

Reply:
Destination is rooted at `out`, 
if you look 5 lines above, we have a similar entry for `pdfkit`